cmd/abapEnvironmentRunATCCheck_test.go

```diff
@@ -24,6 +24,21 @@ func TestHostConfig(t *testing.T) {
 			assert.Equal(t, "", con.XCsrfToken)
 		}
 	})
+	t.Run("Check Host: ABAP Endpoint without HTTPS prefix", func(t *testing.T) {
+		config := abapEnvironmentRunATCCheckOptions{
+			Username: "testUser",
+			Password: "testPassword",
+			Host:     "api.endpoint.com",
+		}
+		var con connectionDetailsHTTP
+		con, error := checkHost(config, con)
+		if error == nil {
+			assert.Equal(t, "testUser", con.User)
+			assert.Equal(t, "testPassword", con.Password)
+			assert.Equal(t, "https://api.endpoint.com", con.URL)
+			assert.Equal(t, "", con.XCsrfToken)
+		}
+	})
```

Comment:
Could you add similar tests to abapEnvironmentPullGitRepo?

Reply:
Done. Can you re-check?